grails-app/views/mobile/inboundList.gsp

```diff
@@ -17,7 +17,7 @@
                     <th><g:message code="supplier.label"/></th>
                     <th><g:message code="stockMovement.deliveryForFa.label" default="Delivery For FA"/></th>
                     <th><g:message code="stockMovement.requestedDeliveryDate.label" default="Requested Delivery Date"/></th>
-                    <th><g:message code="stockMovement.status.label" default="Status"/></th>
+                    <th><g:message code="stockMovement.status.label" default="Status"/>&nbsp;&nbsp;<button class="btn btn-primary"><g:link class="text-white" controller="mobile" action="inboundList" params="[status: 'SHIPPED']">Transit</g:link></button> </th>
```

Comment:
Could you turn the button into a status filter? Just add it right above the table for now. 

For now we’ll just have one status ("In Transit"), but others will be coming.